docs/core-features/table_lookups.md

```diff
@@ -58,11 +61,11 @@ actual_output = circuit.encrypt_run_decrypt(np.array(sample))
 assert np.array_equal(actual_output, expected_output)
 ```
 
-`LookupTable` mimics array indexing in Python, which means if the lookup variable is negative, the table is looked up from the back.
+The `LookupTable` behaves like Python's array indexing, where negative indices access elements from the end of the table.
 
 ## Multi TLU
 
-Multi TLU stands for instructions of the form `y = T[j][i]`, for some set of tables `T`, some table-index `j` and some index `i`. One can use the `fhe.LookupTable` to define the table values, and then use it on scalars or tensors.
+A multi TLU performs operations in the form of `y = T[j][i]`, where `T` is a set of tables, `j` is a table index, and `i` is an element index. You can define the table using `fhe.LookupTable` and apply it to scalars or tensors:
```

Comment:
@umut-sahin : I guess there is no usage to use multi TLU to scalars, no?

Reply:
Nope, it's just for tensors.

Reply:
@umut-sahin to update this please

Reply:
Sure.